### PR TITLE
Tommy Tester's First Focused Reflection added an embed

### PR DESCRIPTION
**Pull request by tommytestertwo**
This is my first pull request that we're testing in front of the class :) We  embedded stuff here

### CODE REVIEW

**Review by eah13 — CHANGES_REQUESTED** [review]
The folder name `tommytester2` doesn't seem to match your github username. Are you sure this is the folder you've been using?

File: _posts/tommytester2/2022-09-29-tommytester-first-focused-reflection.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: tommytester
+title: "Tommy Tester's First Focused Reflection"
```
Comment (eah13) [inline _posts/tommytester2/2022-09-29-tommytester-first-focused-reflection.md:4]: I think you might need to put double quotes around this title

**Review by eah13 — CHANGES_REQUESTED** [review]
I think there might be a problem with your YAML metadata.

File: _posts/tommytester2/2022-09-29-tommytester-first-focused-reflection.md (hunk 1 above)
Comment (tommytestertwo) [reply _posts/tommytester2/2022-09-29-tommytester-first-focused-reflection.md:4]: Hmm. I see `"Tommy Tester's First Focused Reflection"` - am I missing something?

Comment (eah13) [reply _posts/tommytester2/2022-09-29-tommytester-first-focused-reflection.md:4]: Oh, you're right. Apologies.

**Comment by tommytestertwo** [comment]
Thanks for the review! I believe all issues are addressed now. Ready for a hopefully final review.

**Review by eah13 — APPROVED** [review]
Everything looks good!

**Pull request merged**